pkg/inject/inject_test.go

```diff
@@ -431,6 +431,7 @@ func TestPatchGenerator_createInitContainerPatches(t *testing.T) {
 	type fields struct {
 		Strategy           InjectionStrategy
 		Timezone           string
+		InitContainerName  string
```

Comment:
there should be at least one test with different `InitContainerName` that proves it effective

Reply:
sure, will update the test with different `InitContainerName`

Reply:
Hi @yonatankahana, sorry for super late response. I have included the `initContainerName` field and added new test case for different `initContainerName` value